### ISSUE TITLE
Block rule show as pass

### ISSUE THREAD

**Issue by filipdot** [opnsense/core#8104]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [ X ] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [ X ] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Describe the bug**

Versions
OPNsense 24.7.9_1-amd64
FreeBSD 14.1-RELEASE-p6
OpenSSL 3.0.15

Block rule make pass for idk why reason. Block rule block "bad" ip, no mater of port right? That weird.

![obraz](https://github.com/user-attachments/assets/791891ae-5081-49a2-a595-e9a1e7c8a781)

Blocked rule:
![obraz](https://github.com/user-attachments/assets/960d28e9-4c40-4669-a485-19a7c2ecc1d4)

Passed rule:
![obraz](https://github.com/user-attachments/assets/268047ba-91ad-4680-aed7-c3187139c5c3)

If is it true pass but it's blocked and it's wrong in gui or its pass.

Float?
![obraz](https://github.com/user-attachments/assets/8010b195-c3da-4275-bffb-6c1a70773b60)

disabled it on float but it come back on another rule...

![obraz](https://github.com/user-attachments/assets/360c7b88-b532-434f-ac18-752137b3d8b8)

err...
![obraz](https://github.com/user-attachments/assets/5fca4c02-82a4-403c-8eee-439c4f6636d0)

ok look at rule "Nie Polska" with should filter out everything outside Poland IPs using GEOIP
and ip 179.60.147.138 (Venezuela (VE)) looks like outside but it pass... 

![obraz](https://github.com/user-attachments/assets/3f77fd22-5824-4947-a3c2-fa3a9cd70d06)

![obraz](https://github.com/user-attachments/assets/499348e6-5a78-456c-a39d-4b08b2f66e11)

maybe roule? lets go to rid
![obraz](https://github.com/user-attachments/assets/7661eaee-aabe-43e6-9554-2a66a496daad)

looks good, we looking for invert of "Polska"
Alias "Polska":
![obraz](https://github.com/user-attachments/assets/6a38a9e7-9654-4507-8066-aeb52c7bb15b)
![obraz](https://github.com/user-attachments/assets/c6b70371-48a4-41f5-a35d-4582be10b84e)

in csv geolite2 i see match
![obraz](https://github.com/user-attachments/assets/4f6387b7-95d6-4f97-86fa-d6efa172e6bb)

TL DR

why it pass but it should block? Block rule on top. Label and ruleid is from BLOCK rule but it make PASS.

*Tip*: to validate your setup was working with the previous version, use opnsense-revert (https://docs.opnsense.org/manual/opnsense_tools.html#opnsense-revert)
Can't do it.

**To Reproduce**

1. use float rules or wan rules
2. wait for bad connection
3. check live firewall on wan and label block as in rules

**Expected behavior**

all traffic is blocked from "bad" ip

need more?

**Comment by fichtner** [comment]
@filipdot did you check on 24.7.10 or later?
